docs/how-to/explore/find-any-resource.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Find any resource
+sidebar_custom_props:
+  tags: [AWS, GCP, DigitalOcean, Kubernetes]
+---
+
+# How to quickly find any resource
+
+## Introduction
+
+Resoto maintains an asset inventory of your infrastructure metadata. It is equipped with a full-text search index that can be leveraged to find any resource, no matter which cloud provider, account, region or service the resource belongs to.
```

Comment:
Can't put my finger on it, but somehow feels technically incorrect. 

Proposed change:

Resoto builds a cloud asset inventory by collecting metadata from your infrastructure. Resoto is equipped with a full-text search index to discover, monitor, and analyze from a single place - no matter which cloud provider, account, region or service the resource belongs to.

Reply:
Replaced with the proposal.